node/src/proxy_server/http_protocol_pack.rs

```diff
@@ -160,7 +163,8 @@ mod tests {
 
         let host = HttpProtocolPack {}.find_host(&data).unwrap();
 
-        assert_eq!(String::from("header.host.com"), host.name);
+        assert_eq!(host, Host::new("header.host.com", HTTP_PORT));
+        assert_eq!(HTTP_PORT, host.port);
```

Comment:
Now that `Host` has a constructor, we can easily assert on the whole thing, rather than just one field.